azext_iot/central/params.py

```diff
@@ -654,3 +661,261 @@ def load_central_arguments(self, _):
             options_list=["--children-ids"],
             help="Space-separated list of children device ids.",
         )
+
+    with self.argument_context("iot central enrollment-group") as context:
+        context.argument(
+            "group_id",
+            options_list=["--group-id", "--id"],
+            help="Unique identifier for the enrollment group.",
+        )
+        context.argument(
+            "entry",
+            options_list=["--entry"],
+            help="Entry of certificate only support primary and secondary.",
```

Comment:
This is very confusing and makes no sense

Reply:
I'll keep it but add a '--certificate-entry' longer name to this since in some scenarios customers need this parameter to specify which key (primary or secondary) they want to get/verify/remove/generate verification code.